lightning-graph-sync/src/lib.rs

```diff
@@ -0,0 +1,241 @@
+use core::ops::Deref;
+use std::fs::File;
+
+use lightning::chain;
+use lightning::routing::network_graph;
+use lightning_block_sync::http::*;
+
+use crate::error::GraphSyncError;
+
+pub mod error;
+pub mod processing;
+
+pub async fn sync_network_graph_with_url<CS: Deref>(
+    network_graph: &network_graph::NetworkGraph,
+    sync_host: &str,
+    sync_port: u16,
+    sync_path: &str,
+    chain_source: &Option<CS>,
```

Comment:
Lets just drop the `chain_source`, we dont have signatures so its not like we can validate anything.

Reply:
Done